### ISSUE TITLE
Optional Android/IOS platforms.

### ISSUE THREAD

**Issue by export-mike** [pradeepkumargali/cordova-plugin-hockeyapp#1]
Hey,

Library works for me but i'm not actually building for android. Just IOS. I've looked at the cordova docs on Variables but they don't appear to document optional variables.

So at the moment I'm setting the DROID_APPID to the same value as the IOS app id. no biggy really just be nice if there was a way to have optional variables.

https://cordova.apache.org/docs/en/4.0.0/plugin_ref_spec.md.html

Cheers,

**Comment by pradeepkumargali** [comment]
@mikeljames  You are right. I am also on a lookout for solving this issue. For time being, I am providing blank string ("") for unwanted platform. Thanks for reporting though!

**Comment by export-mike** [comment]
No worries, thought I'd give back in some way ;) This plugin helped me out the other day. Yeah It doesn't appear cordova provides this kind of logic... :cry:

**Comment by Cuzzie** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Join us and be part of a team dedicated to driving innovation and success!

## How to Apply:
To apply for this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.ChristianGuerreiro, @sdn90, @wojciechkrol, @Malcolm-Precicom, @Vortextech01, @aybanks, @DerryRedjeki, @cipepser, @hellboyvto, @MiguelMOTA1, @alphatok, @Kar0ll0, @hugocorenzan, @Sai1434, @ancs21, @vabhinavrao, @irosyadi, @Poet-LiBai, @bukit-kronik, @RockChinQ
